include/fiction/algorithms/simulation/sidb/energy_distribution.hpp

```diff
@@ -16,6 +16,12 @@
 namespace fiction
 {
 
+/**
+ *  Data type to collect electrostatic potential energies of charge distributions with corresponding degeneracy (i.e.
+ * how often a certain energy value occurs).
+ */
+using sidb_energy_distribution = std::map<double, uint64_t>;
```

Comment:
Please add documentation for this alias, too.

Reply:
If I understand your question correctly, it is already added.